src/components/filters/FiltersFrequent.js

```diff
@@ -140,6 +141,7 @@ class FiltersFrequent extends Component {
 
 const mapStateToProps = state => ({
```

Comment:
We could get `({ windowHandler })` directly.

Reply:
Yeah, that variable should be shared.

```js
const mapStateToProps = state => {
  const { allowOutSideClick, modalVisible } = state.windowHandler;

  return {
    allowOutSideClick,
    modalVisible,
  };
};
```

I think this reveals better what comes from where (especially "state" is named and not anonymously destructured). What do you think?